Makefile

```diff
@@ -112,14 +116,14 @@ $(TOPDIR)/bin: $(TOPDIR)
 
 # Linux AMD64 Builds
 $(TOPDIR)/bin/amd64-linux/habitat: $(TOPDIR)/bin frontend_server/build
-	GOARCH=amd64 GOOS=linux go build -o $(TOPDIR)/bin/amd64-linux/habitat $(TOPDIR)/cmd/node/main.go
+	GOARCH=amd64 GOOS=linux go build -o $(TOPDIR)/bin/amd64-linux/habitat $(TOPDIR)/cmd/node/main.go && go build -o $(TOPDIR)/bin/amd64-linux/node-ctrl $(TOPDIR)/cmd/node_ctrl/main.go
```

Comment:
This targetis a file path, so it's main artifact should just be the bin compiled to `$(TOPDIR)/bin/amd64-linux/habitat`. IIRC currently this rule is only used by the release script. If you want to include the CLI in the release, we can just add the target for the CLI bin as a prerequisite to the main `build` rule.